lib/models/package-info-cache/index.js

```diff
@@ -12,91 +12,10 @@ const Errors = require('./errors');
 const PackageInfo = require('./package-info');
 const NodeModulesList = require('./node-modules-list');
 const logger = require('heimdalljs-logger')('ember-cli:package-info-cache');
+const resolvePackageCache = require('resolve-package-path');
```

Comment:
Suggest calling this resolvePackagePath instead, just to match the package name.